src/Faker/Provider/HtmlLorem.php

```diff
@@ -66,6 +66,36 @@ public function randomHtml($maxDepth = 4, $maxWidth = 4)
         return $document->saveHTML();
     }
 
+    /**
+     * @param int $maxDepth
+     * @param int $maxWidth
+     *
+     * @return string
+     */
+    public function randomBody($maxDepth = 4, $maxWidth = 4)
```

Comment:
Now that I'm using this in my actual developing code, I think this would be better called `randomHTMLBody`.